awx/__init__.py

```diff
@@ -149,6 +150,12 @@ def manage():
     from django.conf import settings
     from django.core.management import execute_from_command_line
 
+    # enforce the postgres version is equal to 12. if not, then terminate program with exit code of 1
```

Comment:
This comment should probably say `equal to or greater than` to reflect the code.  Newer versions of postgresql for external db's shouldn't be an issue, though it isn't strictly supported.